script/dump_book.py

```diff
@@ -0,0 +1,399 @@
+#!/var/cnx/venvs/archive/bin/python
+"""
+This script attempts to dump a book from the server and load it into a
+development database.
+
+1. You need to download this script onto the server, e.g. staging08.cnx.org,
+   that runs archive.
+
+2. Use `./dump_book.py dump_book caa57dab-41c7-455e-bd6f-f443cda5519c@19.3` to
+   dump the prealgebra book.
+
+   2.1 If you are not using this on a cnx-deploy'd server, you can either
+       specify the path to the archive config file using `CONFIG_INI` or the
+       database connection string using `DB_CONN_STR` environment variables.
+
+   2.2 If you want to dump multiple books, just add more book ident hashes to
+       the end of the command line.
+
+3. Transfer the output file
+   `book_dump.prealgebra@19.3.tar` to the
+   development machine.
+
+4. Run `./dump_book.py load_book book_dump.prealgebra@19.3.tar`
+   to load the book into a development database.
+
+   4.1 If you are not using this on a cnx-deploy'd server, you can either
+       specify the path to the archive config file using `CONFIG_INI` or the
+       database connection string using `DB_CONN_STR` environment variables.
+
+   4.2 If you want to load multiple files, just add the filenames to the end of
+       the command line.
+
+**Note**: strictly for development use only.
+"""
+
+import base64
+try:
+    import ConfigParser as configparser
+except ImportError:
+    import configparser
+import contextlib
+import json
+import logging
+import os
+import re
+import shutil
+import socket
+import sys
+import tarfile
+import tempfile
+
+import psycopg2.extras
+
+
+DB_CONN_STR = os.getenv('DB_CONN_STR')
```

Comment:
I'm guessing it's a little late to go with `DB_URL`? `DB_URL` is used by the `cnx-db` script as well as other parts of the logic within Press. It's no big deal. Just mentioning it.

Reply:
That's totally fine, I'll change it.  I don't actually use this environment variable because I use the config ini.